### PR TITLE
Adjust to catkin_tools 0.4.0

### PR DESCRIPTION
**Pull request by 130s**
Initialized by #30, with the Travis build of this PR I'm trying to find if there's any difference introduced by [catkin_tools 0.4.0 that was released today](http://lists.ros.org/pipermail/ros-users/2016-April/069968.html).

### CODE REVIEW

**Comment by 130s** [comment]
Travis tests are all green.
I'm inclined to merge this and see how #30 goes, but I need +1 to do so.

**Comment by VictorLamoine** [comment]
:+1: I came up with the exact same fix

**Comment by 130s** [comment]
@VictorLamoine thanks!

This should fix #30.

File: package.xml [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
   <build_depend>catkin</build_depend>
   <build_depend>python</build_depend>
+  <build_depend version_gte="0.4.0">python-catkin-tools</build_depend>
+  <run_depend version_gte="0.4.0">python-catkin-tools</run_depend>
```
Comment (ipa-mdl) [inline package.xml]: If I remember it correctly, the version are ignored by rosdep and are just for the release packaging.
However, they are good indicators for people that are debugging problems.

Comment (130s) [inline package.xml]: Yeah, I guess only systematic reaction is that catkin [prints warning in a certain condition](http://answers.ros.org/question/136526/how-versioned-build_depend-works/). 
I added it more for human purpose, not for machine.

**Pull request merged**